src/app/(app)/archive/page.jsx

```diff
@@ -63,10 +70,25 @@ export default function Home() {
             </EntryMonthWrapper>
           );
         })}
-        <div className={styles.centerContainer}>
-          {entryList.length < totalEntries && <button onClick={fetchMoreEntries}>Show more</button>}
-        </div>
-      </div>
+        {loading && (
+          <div className={styles.centerContainer}>
+            <RotatingLines strokeColor='#f3b04e' height='48' width='48' />
```

Comment:
Not a huge fan of inline hexcode. Should we think about abstracting this component to standardize the icon across the board?